romancal/tweakreg/tweakreg_step.py

```diff
@@ -47,10 +46,9 @@ class TweakRegStep(RomanStep):
     class_alias = "tweakreg"
 
     spec = f"""
-        save_catalogs = boolean(default=False) # Write out catalogs?
-        use_custom_catalogs = boolean(default=False) # Use custom user-provided catalogs?
-        catalog_format = string(default='ecsv') # Catalog output file format
-        catfile = string(default='') # Name of the file with a list of custom user-provided catalogs
+        save_catalog = boolean(default=False) # Write out the input catalog?
```

Comment:
Is it necessary to change the name of the parameter (compared to jwst.tweakreg)? 
`save_catalogs` -> `save_catalog`

Reply:
This option would allow the user to save the source catalog created by `SourceDetection`.
Since that option is already available in `SourceDetectionStep`, we removed it from `TweakRegStep`.